server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -1036,13 +1051,38 @@ func (s *BuildBuddyServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// TODO(siggisim): Figure out why this JWT is overriding authority auth and remove.
+	ctx := context.WithValue(r.Context(), "x-buildbuddy-jwt", nil)
+
+	var archiveReference = params.Get("z")
+	if len(archiveReference) > 0 {
+		b, err := base64.StdEncoding.DecodeString(archiveReference)
+		if err != nil {
+			log.Warningf("Error downloading file: %s", err.Error())
```

Comment:
we are often lazy and omit the ".Error()" because it will work fine without that

Reply:
(it's inconsistent, and this is older code so probably wasn't as common then)

Reply:
Fixed this and the one below, too.